test/e2e/next-test-lib/next-modes/next-deploy.ts

```diff
@@ -28,22 +39,23 @@ export class NextDeployInstance extends NextInstance {
       cwd: this.testDir,
       stdio: 'inherit',
     })
-    // ensure Netlify CLI is installed
+    // Netlify CLI should be installed, but just making sure
     try {
       const res = await execa('ntl', ['--version'])
       console.log(`Using Netlify CLI version:`, res.stdout)
     } catch (_) {
-      console.log(`Installing Netlify CLI`)
-      await execa('npm', ['i', '-g', 'netlify-cli@latest'], {
-        stdio: 'inherit',
-      })
+      throw new Error(`You need to have netlify-cli installed.
+      
+      You can do this by running: "npm install -g netlify-cli@latest" or "yarn global add netlify-cli@latest"`)
     }
 
-    const NETLIFY_SITE_ID = process.env.NETLIFY_SITE_ID || '1d5a5c76-d445-4ae5-b694-b0d3f2e2c395'
     console.log(`Deploys site for test: ${testName}`)
+
+    this._netlifySiteId = process.env.NETLIFY_SITE_ID || '1d5a5c76-d445-4ae5-b694-b0d3f2e2c395'
```

Comment:
What's the default site ID `'1d5a5c76-d445-4ae5-b694-b0d3f2e2c395'`?

Reply:
Who knows... 😆 That code was already there, I only moved it